script/check_chromium_src.py

```diff
@@ -47,6 +47,7 @@
     'content/browser/tld_ephemeral_lifetime.cc',
     'content/public/browser/tld_ephemeral_lifetime.h',
     'third_party/blink/common/origin_trials/origin_trials.cc',
+    'third_party/blink/renderer/bindings/modules/v8/v8_navigator.cc',
     'third_party/blink/renderer/modules/battery/navigator_batterytest.cc',
     'third_party/blink/renderer/modules/bluetooth/navigator_bluetoothtest.cc',
     'third_party/blink/renderer/modules/quota/navigator_storagetest.cc',
```

Comment:
I've moved some of these files (tests) to appropriate places, so this list can be cleaned-up.

Reply:
Fixed in https://github.com/brave/brave-core/commit/7e05780d2969ee10a00635406240ecd9f68206fb